changelogs/fragments/64294-yum-handle-invalid-nevra.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - dnf - fixed an issue where invalid NEVRA values in package specs would fail to remove/uninstall
```

Comment:
```suggestion
  - dnf - fixed an issue where invalid NEVRA values in package specs would fail to remove/uninstall (https://github.com/ansible/ansible/issues/64294)
```